docs/stack/insight/tutorials/insight-stream-consumer.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Manage streams and consumer groups in RedisInsight"
+linkTitle: Streams
+description: Learn how to manage streams and consumer groups in RedisInsight
+weight: 1
+---
+
+A _stream_ is an append-only log file. 
+When you add data to it, you cannot change it. 
+That may seem like a disadvantage; however, a stream serves as a log or single source of truth. 
+It is used as a buffer between processes that work at different speeds and do not need to know about each other. 
+For more conceptual information about streams, see [Redis Streams](/docs/manual/data-types/streams). 
+
+In this topic, you will learn how to add and work with streams as well as consumer groups in RedisInsight.
+
+Here's a stream that models temperature and humidity sensors. 
+In this stream, sensors are considered _producers_, which broadcast data. 
+The point of a stream is that it's not going to end, so you cannot capture whole datasets and do some processing on them. 
+In order to process the data, a stream is separated into different roles. 
```

Comment:
```suggestion
Processes interacting with the stream perform one of two roles:```

The two roles are "consumer" and "producer".  I think there needs to be a sentence here about the producer role before we go into the consumer role?